src/Build/BuildCheck/OM/ProjectImportedCheckData.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Build.Experimental.BuildCheck;
+
+namespace Microsoft.Build.Experimental.BuildCheck;
+
+public class ProjectImportedCheckData : CheckData
+{
+    public ProjectImportedCheckData(string importedProjectFile, string projectFilePath, int? projectConfigurationId)
+        : base(projectFilePath, projectConfigurationId) => ImportedProjectFile = importedProjectFile;
+
+    public string ImportedProjectFile { get; }
```

Comment:
```suggestion
    public string ImportedProjectFilePath { get; }
```

Btw. is it a full path?
It might be helpful to have this as a full path - and then name it so (as it might not be that easy for consumers to properly reconstruct the full path)

Reply:
Is it guaranteed to be a full path?
If not - can we make it so?

Reply:
from what I have observed on the check run, only full paths were reported.
@rainersigwald , are you aware of any edge cases ?

Reply:
Looking on all the usages of [ProjectImportedEventArgs.ImportedProjectFile.set](https://github.com/dotnet/msbuild/blob/5881e053656bc8aaa890515e5d4318af39f49dc5/src/Framework/ProjectImportedEventArgs.cs#L60) it gets a full path (unless import was skipped - which we should be skipping here anyways)

So it can be renamed to `ImportedProjectFileFullPath`